Makefile

```diff
@@ -18,12 +18,12 @@ format: node_modules
 
 .PHONY: clean
 clean:
-	rm -rf node_modules styleguide-app/elm.js styleguide-app/bundle.js $(shell find . -type d -name 'elm-stuff')
+	rm -rf node_modules styleguide-app/elm.js styleguide-app/bundle.js $(shell find . -type d -name 'elm-stuff') public
 
 .PHONY: styleguide-app
 styleguide-app: styleguide-app/elm.js
-	@echo "Visit http://localhost:8000/index.html to see the styleguide app in your browser"
-	cd styleguide-app && npx elm reactor
+	@echo "the styleguide-app command is deprecated; use serve-public."
```

Comment:
💭 Why not remove it?

Reply:
Because I bet a few people say `make styleguide-app` and it would be confusing if that was just not a thing anymore without any reason. This way seems more discoverable in that case.

Reply:
I wonder if the new thing could be called styleguide-app, in that case?